### PR TITLE
Update translations (credits, french).

### PR DESCRIPTION
**Pull request by Kirvx**

### CODE REVIEW

**Comment by JustinDrake** [comment]
+1 good translations.

One comment. In "to make trade free", the word 'free' is somewhat overloaded. It means both free as in freedom, and free as in free of charge. My suggestion:

"Vous pouvez aider OpenBazaar de plusieurs manières afin de faciliter les échanges commerciaux **libres et** gratuits."

**Comment by Kirvx** [comment]
Done :)

File: js/models/languagesMd.js [hunk 1]
```diff
@@ -6407,10 +6541,21 @@ module.exports = Backbone.Model.extend({
...
       {
+        
         langName: "Polski",
         langCode: "pl",
-        /* Use capitalized keys for widely reused text that must be capitalized */
+        
+        /*
+        
+        Translator: @Mido
+        02/18/2016: ~ 71 untranslated strings
```
Comment (hoffmabc) [inline js/models/languagesMd.js:6558]: I like this addition, but I just want to make sure that we all are able to keep this data in sync as it's not automated at present (how many strings untranslated that is).

File: js/models/languagesMd.js [hunk 2]
```diff
@@ -6407,10 +6541,21 @@ module.exports = Backbone.Model.extend({
...
+        
+        Use capitalized keys for widely reused text that must be capitalized
```
Comment (hoffmabc) [inline js/models/languagesMd.js:6560]: We should probably have these designated terms available somewhere for consistency.

**Comment by hoffmabc** [comment]
@Kirvx there seem to be some conflicts in your Pull Request. Please rebase your branch or resolve the conflicts and update. That way we can merge your changes. Thanks!

**Comment by Kirvx** [comment]
Updated.
Added credits for Danish and Chinese, and updated untranslated strings for Chinese.

**Pull request merged**